nexus/bin/qdens-radial

```diff
@@ -0,0 +1,513 @@
+                    self.error('Input species (-S,--species) must be a subset of the species in the .xsf file.\nThe .xsf contains: {}\nYou provided: {}.'.format(equiv_atoms,opt.species.tolist()))
+                #end if
+                if opt.cumulative:
+                    if opt.species is None:
+                        rdens_samp = cds.cumulative_radial_density(rmax=opt.radii.tolist())
+                    else:
+                        rdens_samp = cds.cumulative_radial_density(rmax=opt.radii.tolist(),species=opt.species.tolist())
+                    #end if
+                else:
+                    if opt.species is None:
+                        rdens_samp = cds.radial_density(rmax=opt.radii.tolist())
+                    else:
+                        rdens_samp = cds.radial_density(rmax=opt.radii.tolist(),species=opt.species.tolist())
+                    #end if
+                #end if
+
+                if opt.species is not None:
+                    for ati,at in enumerate(opt.species):
+                        sum_dict[at] += rdens_samp.tot[at].density[-1]
+                        sum_sq_dict[at] += rdens_samp.tot[at].density[-1]**2.0
+                    #end for
+                else:
+                    for ati,at in enumerate(equiv_atoms):
+                        sum_dict[at] += rdens_samp.tot[at].density[-1]
+                        sum_sq_dict[at] += rdens_samp.tot[at].density[-1]**2.0
+                    #end for
+                #end if
+
+            #end for
+            if opt.species is not None:
+                for ati,at in enumerate(opt.species):
+                    std_dict[at] = np.sqrt(sum_sq_dict[at]/nsamples-(sum_dict[at]/nsamples)**2.0)
+                #end for
+            else:
+                for ati,at in enumerate(equiv_atoms):
+                    std_dict[at] = np.sqrt(sum_sq_dict[at]/nsamples-(sum_dict[at]/nsamples)**2.0)
+                #end for
+            #end if
+
+        else:
+            if opt.vmc_err_file is not None:
+                self.error('VMC and DMC densities provided but only one +err file was given.\nPlease either provide no +err file (which avoids resampling) or\nprovide a +err file for both VMC and DMC with --vmc_err_file and --dmc_err_file arguments.')
+            #end if
+        #end if
+
+        # Instantiate ChargeDensity object
+        cd = ChargeDensity()
+        cd.read_xsf(file)
+        equiv_atoms = list(cd.info.structure.equivalent_atoms().keys())
+
+        if opt.species is not None and not set(opt.species).issubset(equiv_atoms):
+            self.error('Input species (-S,--species) must be a subset of the species in the .xsf file.\nThe .xsf contains: {}\nYou provided: {}.'.format(equiv_atoms,opt.species.tolist()))
+        #end if
+
+        if opt.source=='pwscf':
+            cd.change_distance_units('B')
+        else:
+            cd.volume_normalize()
+        #end if
+
+        self.log('\nNorm:',cd.norm())
+
+        if opt.plot:
```

Comment:
Should plot cumulative density instead or in addition w/ `-c` requested?

Reply:
Resolved.

Reply:
Now plotting cumulative density with `-c` requested.